content/author/manjot-sidhu.md

```diff
@@ -0,0 +1,29 @@
++++
+title = "Manjot Singh Sidhu"
```

Comment:
You can either update Manjot Sidhu here or in content/posts/Manjot_gsoc_phase_one_2021.md author -> Manjot Singh Sidhu, as the blog is not able to recognise the blogs written by you on the author's page

Reply:
Sure